pkgs/development/python-modules/sismic/default.nix

```
@@ -0,0 +1,52 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  setuptools,
+  behave,
+  ruamel-yaml,
+  schema,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "sismic";
+  version = "1.6.6";
+  build-system = [ setuptools ];
```

```suggestion
  version = "1.6.6";
  pyproject = true;

  build-system = [ setuptools ];
```
Please add `pyproject = true` to avoid the legacy setuptools build.
> pyproject: Whether the pyproject format should be used. 
...
When unset, the legacy setuptools hooks are used for backwards compatibility.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

---

Fixed.